### DISCUSSION TITLE
Why CycleGAN uses only 1 sample per gpu？

### DISCUSSION THREAD

**Discussion by jiehuang165** [open-mmlab/mmediting#1361] [How-to] [answered]
All CycleGAN's [configs](https://github.com/open-mmlab/mmediting/tree/master/configs/synthesizers/cyclegan) use `samples_per_gpu=1`, is there some particular reasons or just limited by GPU memory?

**Comment by LeoXing1996** [comment, marked as answer]
The implementation of CycleGAN refers to [the official implementation](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/e2c7618a2f2bf4ee012f43f96d1f62fd3c3bec89/options/base_options.py#L45), which sets `batch_size` as 1.
